src/screens/FarmSearch/components/FarmCard.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { Component } from 'react';
+import Modal from 'react-modal';
+
+import '../../../styles/FarmCard.css';
+
+const ROOT_ELEMENT = '#root';
+Modal.setAppElement(ROOT_ELEMENT);
+
+class FarmCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showFarmModal: false
+    };
+  }
+
+  showModal = () => {
+    this.setState({ showFarmModal: true });
+  };
+
+  hideModal = () => {
+    this.setState({ showFarmModal: false });
+  };
+
+  render() {
+    const { showFarmModal } = this.state;
+    const {
+      hubApproved,
+      gapApproved,
+      farmName,
+      farmerName,
+      farmLocation,
+      farmCover,
+      userIcon
+    } = this.props;
+
+    return (
+      <div className="farm-card">
+        <div className="farm-card__top">
+          <img
+            src={farmCover}
+            alt="defaultfarmCover"
+            className="farm-card__photo"
+            onClick={this.showModal}
```

Comment:
change this to show farm profile based on auth

Reply:
Gonna move this specific feature to a new PR